src/components/presentation/Presentation.tsx

```diff
@@ -30,14 +39,22 @@ export default async function Presentation({ presentation, isFrontPage }: Presen
               </Link>
             </h3>
           )}
-          {!isFrontPage && <h1 className='mb-8 hyphens-auto sm:hyphens-none'>{title}</h1>}
+          {!isFrontPage && (
+            <div>
+              <h1 className='mb-2 hyphens-auto sm:hyphens-none'>{title}</h1>
+            </div>
+          )}
           <div className='flex flex-col md:flex-row gap-8'>
             {!isFrontPage && (
-              <p className='text-stone-200 text-base sm:text-[20px] whitespace-pre-line'>{description}</p>
+              <div>
+                <p className='mb-8 text-[25px]'>{`${presentation.room}${time}`}</p>
+                <p className='text-stone-200 text-base sm:text-[20px] whitespace-pre-line'>{description}</p>
+              </div>
```

Comment:
I think it would be better if the time info would be in the same div as the title. This way it's right under the title even on mobiles. It looks a bit weird between the image and the description.

Reply:
The side image was too far down like that for me. But it should be fixed now.